libs/encryption-sdk/build.gradle

```diff
@@ -19,13 +19,8 @@ dependencies {
   // Common crypto classes
   api project(':libs:opensearch-common')
 
-  // Logger
-  implementation "org.slf4j:slf4j-api:${versions.slf4j}"
-  implementation 'commons-logging:commons-logging:1.2'
-
   // Encryption
   implementation "com.amazonaws:aws-encryption-sdk-java:2.4.0"
-  implementation "org.bouncycastle:bcprov-jdk15to18:${versions.bouncycastle}"
   implementation "org.apache.commons:commons-lang3:${versions.commonslang}"
```

Comment:
I don't see any imports from commons-lang3 in this library. Is this something that can be removed too or is it needed?

Reply:
That's the transitive one for `aws` :( 

```
compileClasspath - Compile classpath for source set 'main'.
+--- project :libs:opensearch-common
+--- com.google.code.findbugs:annotations:3.0.1
+--- com.amazonaws:aws-encryption-sdk-java:2.4.0
\--- org.apache.commons:commons-lang3:3.13.0
```

Reply:
@reta But this causing downstream plugins to break. :( https://github.com/opensearch-project/neural-search/actions/runs/6094181033/job/16535187721?pr=287

Reply:
@navneet1v I know :( but we a) either completely drop the  `encryption-sdk` or b) the `neural-search` could just exclude the common-lang3 from dependencies, the sql plugin has same issue

Reply:
@navneet1v This should get auto-fixed once https://github.com/opensearch-project/OpenSearch/pull/9833 or https://github.com/opensearch-project/OpenSearch/pull/9812 is merged

Reply:
@vikasvb90 thanks :)